apps/api/src/schema/lib/year.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from "@hono/zod-openapi";
+
+/**
+ * Confirms string for year is in valid format
+ * Must be 4 characters
+ * Could be String/Integer upon confirmation
+ *
+ * 1. Checks if it is a string
+ * 2. Checks length of string is 4
+ * 3. Checks if it is an integer
```

Comment:
Same as above; this is not actually what the parsing does! A less careful reader might not realize the distinction and trust your comment, relying on an incorrect understanding and writing unsound code based on it.